app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,191 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+$light_gray: #F2F2F2;
+$white: #FFF;
+$beach_blue: #5bc0de;
+
+body {
+  padding-top: 20px;
+}
+
+#index_background_header {
+  text-align: center;
+  padding-top: 300px;
+  font-family: Circular,"Helvetica Neue",Helvetica,Arial,sans-serif;
+  color: #FFF;
+  font-size: 5em;
+  font-weight: bold;
+}
+#search_bar_container .well{
+  background: #000;
+  background: rgba(0,0,0,.1);
+  position: absolute;
+  bottom: 0;
+  margin-bottom: 0;
+  left: 0;
+  right: 0;
+  border-color: transparent;
+}
+#index_image_one {
+  background: url("http://photos.cntraveler.com/2014/07/31/53da72466dec627b149ead94_la-digue-island-seychelles-Arco-images-GmbH-Alamy.jpg");
+  background-size: cover;
+  height: 400px;
+}
+#index_image_two {
+  background: url("http://www.vosizneias.com/wp-content/uploads/2015/06/miamb.jpg");
+  background-size: cover;
+  height: 400px;
+}
+#index_image_three {
+  background: url("http://dsx.weather.com//util/image/w/bestbeaches_lumahaihawaii.jpg?v=at&w=980&h=551&api=7db9fe61-7414-47b5-9871-e17d87b8b6a0");
+  background-size: cover;
+  height: 400px;
+}
+.columns {
+  padding-top: 150px;
+}
+.index_containers {
+  margin-top: 100px;
+}
+.index_containers > * {
+  color: #FFF;
+}
+.index_containers > h1 {
+  color: inherit;
+}
+.index_containers > p {
+  color: inherit;
+}
+
+#beach_snapshot {
+  span {
+    color: darken($light_gray, 30%);
+    padding-bottom: 100px;
+  }
+  h1 {
+    margin-bottom: 3px;
+  }
+  margin-bottom: 15px;
+}
+
+#beach_snapshot_icon_container {
+  margin-top: 25px;
+}
+
+.center {
+  text-align: center;
+}
+
+.inline_block {
+  display: inline-block;
+}
+
+.amenity_name {
+  vertical-align: 9px;
+}
+
+.col-centered {
+    display:inline-block;
+    float:none;
+    text-align:center;
+}
+
+.section_title {
+  display: inline-block;
+  vertical-align: 20px;
+}
+
+footer {
+  background: $beach_blue;
+  a, span {
+    color: $white;
+    opacity: .8;
+  }
+  a:hover {
+    color: $white;
+    text-decoration: none;
+  }
+  h3 {
+    color: $white;
+  }
+}
+
+hr {
+  border-color: lighten(gray, 30%);
+}
+
+#popular_beaches_container a, .beach_container a {
+  color: $white;
+  &:hover {
+    text-decoration: none;
+    opacity: .9;
+  }
+}
+
+.beach_container {
+  margin-bottom: 50px;
+}
+
+.beach_index_location_header {
+  color: white;
+  opacity: .7;
+  font-size: .6em;
+}
+
+h3.text-muted {
+  margin-top: 0;
+}
+
+.text-muted a {
+  color: $beach_blue;
+  &:hover {
+    text-decoration: none;
+    color: lighten($beach_blue, 20%);
+  }
+}
+
+div.header.clearfix {
+  margin-bottom: 20px;
+}
+
+.nav.nav-pills.pull-right li a {
+  color: $beach_blue;
+}
+
+#create_a_beach_btn a {
+  color: $white;
+  border: none;
+  &:hover {
+    background: $beach_blue;
```

Comment:
Name of variable `beach_blue` should be written in all lowercase letters with hyphens instead of underscores